packages/api-rx/README.md

```diff
@@ -0,0 +1,49 @@
+[![polkadotjs](https://img.shields.io/badge/polkadot-js-orange.svg?style=flat-square)](https://polkadot.js.org)
+![isc](https://img.shields.io/badge/license-ISC-lightgrey.svg?style=flat-square)
+[![style](https://img.shields.io/badge/code%20style-semistandard-lightgrey.svg?style=flat-square)](https://github.com/Flet/semistandard)
+[![npm](https://img.shields.io/npm/v/@polkadot/api-observable.svg?style=flat-square)](https://www.npmjs.com/package/@polkadot/api-rx)
```

Comment:
should this be `[![npm](https://img.shields.io/npm/v/@polkadot/api-rx.svg` instead of `[![npm](https://img.shields.io/npm/v/@polkadot/api-observable.svg`